src/springcraftNMA.py

```diff
@@ -0,0 +1,209 @@
+# This file is used to calculate/extract and plot normal modes from a file or from PDBID
+
+import biotite
+import biotite.structure as struc
+import biotite.structure.io.mmtf as mmtf
+import biotite.database.rcsb as rcsb
+import biotite.structure.io as strucio
+import springcraft
+import scipy
+import torch
+import numpy as np
+import matplotlib.pyplot as plt
+from typing import Literal
+
+def calculate_normal_modes_gnm_from_file(path, cutoff):
```

Comment:
Same as in other files: Add type hints, run black formatting, rename file to comply with standard python conventiosn